ts_export/src/process.rs

```diff
@@ -190,6 +193,7 @@ impl<PS, E> Process<PS, E>
 where
     PS: ProcessSpawner,
     E: Exporter,
+    TsExportError: From<PS::Error>,
```

Comment:
Not 100% sure if this the best to have this boundary.
Maybe `TsExportError` should have `Custom` invariant or something.

Reply:
In the long run, we may want to support something else than just `TsExportError`, and so this bound will become more relevant. I think this is the correct call.